packages/api-reference-editor/tsconfig.json

```diff
@@ -1,9 +1,13 @@
 {
   "extends": "../../tsconfig.json",
+  "include": ["src"],
+  "exclude": ["dist", "vite.config.ts", "node_modules"],
```

Comment:
We don't need to exclude anything that is not included.

Reply:
what things are not included? also this is outdated its 

```
"exclude": ["dist", "node_modules"],
```

Reply:
Yeah, but then it’s already there when we remove rootDir. :)

Reply:
@marclave include say "src", so vite…, node_modules and dist isn’t included (yet).

Reply:
@hanspagel ah no i meant want things do not need to add to the `exclude` array

Reply:
include only includes src/ and src/ doesn't have vite.config.ts, node_modules and dist, so there’s no point in excluding the files that are one level higher than src.